mleap-runtime/src/main/scala/ml/combust/mleap/runtime/transformer/feature/OneHotEncoderV23.scala

```diff
@@ -0,0 +1,64 @@
+package ml.combust.mleap.runtime.transformer.feature
+
+import ml.combust.mleap.core.feature.OneHotEncoderModel
+import ml.combust.mleap.core.types._
+import ml.combust.mleap.runtime.frame.{MultiTransformer, Row, Transformer}
+import ml.combust.mleap.runtime.function.UserDefinedFunction
+import ml.combust.mleap.core.util.VectorConverters
+
+case class OneHotEncoderV23(override val uid: String =
+                              Transformer.uniqueName("one_hot_encoder_v23"),
+                            override val shape: NodeShape,
+                            override val model: OneHotEncoderModel)
+    extends MultiTransformer {
+  override val exec: UserDefinedFunction = {
+    // UGLY: I don't like this.  It's the best way forward from my current vantage point though.
+    // This stems from the fact that anonymous functions in scala cannot have varargs parameters.
+    shape.inputs.keys.size match {
```

Comment:
Can return a Row(values: _*) for multiple outputs.

Reply:
Take a look here for an example. Will be good too, because then we can support any number of outputs:

https://github.com/combust/mleap/blob/master/mleap-runtime/src/main/scala/ml/combust/mleap/runtime/transformer/classification/DecisionTreeClassifier.scala#L16-L42

Reply:
I'm already spreading the results of the model (see L23) as input to `Row`.  The problem is with not being able to take vararg inputs.

Reply:
Ah, my bad. Here is example of vararg inputs:
https://github.com/combust/mleap/blob/master/mleap-runtime/src/main/scala/ml/combust/mleap/runtime/transformer/feature/VectorAssembler.scala#L25